netlogo-gui/src/main/editor/EditorMenu.scala

```diff
@@ -0,0 +1,9 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.editor
+
+import javax.swing.Action
+
+trait EditorMenu {
+  def offerAction(action: Action): Unit
```

Comment:
That makes sense to me, but it makes me ask: "in what situations will a menu decide not to add a menu entry?"